docs/sources/mimir/references/configuration-parameters/index.md

```diff
@@ -564,10 +564,14 @@ grpc_tls_config:
 # CLI flag: -server.log-source-ips-regex
 [log_source_ips_regex: <string> | default = ""]
 
+[log_request_headers: <boolean> | default = ]
+
 # (advanced) Optionally log requests at info level instead of debug level.
 # CLI flag: -server.log-request-at-info-level-enabled
 [log_request_at_info_level_enabled: <boolean> | default = false]
 
+[log_request_exclude_headers_list: <string> | default = ""]
```

Comment:
could go in changelog; some users will like the ability to restrict more headers

Reply:
I'm adding a PR in weaveworks common to add the flag description as well.